### PR TITLE
Require users to be logged in to view the VICE admin page

### PR DESCRIPTION
**Pull request by johnworth**
Don't worry, the VICE admin page wasn't accessible before to logged out users, it just errored out strangely rather than give good feedback. This PR makes it show the same error component used elsewhere in Sonora when a logged out user tries to access the page.

### CODE REVIEW

File: src/components/vice/admin/index.js [hunk 1]
```diff
@@ -4,12 +4,9 @@ import { useTranslation } from "i18n";
 import { useQuery, useMutation, queryCache } from "react-query";
 
 import { makeStyles } from "@material-ui/styles";
+import WrappedErrorHandler from "../../utils/error/WrappedErrorHandler";
```
Comment (sri2k1us) [inline src/components/vice/admin/index.js:7]: Not an issue at all but reminder that we can use absolute imports - `components/utils/error/WrappedErrorHandler`.

**Review by sri2k1us — APPROVED** [review]
LGTM. :shipit:

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

**Review by psarando — APPROVED** [review]
LGTM :shipit:

**Pull request merged**